mkosi/__init__.py

```diff
@@ -2493,8 +2493,34 @@ def invoke_apt(
         INITRD="No",
     )
 
-    run_workspace_command(args, root, cmdline, network=True, env=env)
+    # Ensure apt inside the image can see the local mirror outside of it
+    if args.mirror and args.mirror.startswith("file:"):
```

Comment:
I'd really prefer a new --bind option to bind mount directories from the host into the container when building instead of doing it this way.

(Or someone finally switches us to mmdebstrap so we don't have to run apt inside the container anymore)

Reply:
But then you need to pass different options depending on the type of mirror? It seems much better to me to do it this way, as it's simpler for users - it's trivial to detect local vs remote (file:) so doing the right thing automatically is nice and easy

Reply:
and also this would be a specific option for debian/ubuntu builds, which is not nice either (I need to use the same arguments for every build in OBS)

Reply:
I agree with @bluca here. A general bind option would be neat, but I think that's a bit beyond this PR. Because either `--local-mirror` would still be needed for what it does here or the whole rigamarole would need to be implemented as some mix of build and postinst scripts, which would be cumbersome for those users.

Also, is GH acting up for me or is the review for an older version of the PR?